### PR TITLE
al_get_display_refresh_rate fix under wgl

### PR DESCRIPTION
**Pull request by aldrikboy**
set refresh_rate for ALLEGRO_FULLSCREEN, ALLEGRO_FULLSCREEN_WINDOW and windowed mode under wgl

fix issue #1165

### CODE REVIEW

File: include/allegro5/monitor.h [hunk 1]
```diff
@@ -16,6 +16,7 @@ typedef struct ALLEGRO_MONITOR_INFO
    int y1;
    int x2;
    int y2;
+   int refresh_rate;
```
Comment (SiegeLord) [inline include/allegro5/monitor.h:19]: Sadly we can't do this as this breaks backwards compatibility. It'd need to be a new function `al_get_monitor_refresh_rate`.

Also, if we're using it just in the windows backend (and I see you've only implemented it on Windows), it should be a private function for now.

Comment (aldrikboy) [reply include/allegro5/monitor.h]: monitor refresh rate can now be retrieved with al_get_monitor_refresh_rate

**Comment by SiegeLord** [comment]
(just FYI, I'm just waiting on getting the time to figure out how well this can be implemented on Linux/OSX etc)

**Comment by SiegeLord** [comment]
Whoops, I got confused, I didn't mean to merge this quite yet.

**Pull request merged**